api/controllers/document.js

```diff
@@ -416,7 +416,7 @@ exports.protectedOpen = function (args, res, next) {
       if (data && data.length === 1) {
         var blob = data[0];
 
-        console.log('the doc', data)
+        defaultLog.info('the doc', data)
```

Comment:
Yeah, I'd say you're right here. I'm trying to take steps toward better logging in the app. For this PR, since a full logging overhaul is out of scope, I just blanket replaced console.logs and console.error with the equivalent defaultLog method. A future ticket will look at our logging policy for the app and refine. Thanks for the flag